src/modules/logger/logger.cpp

```diff
@@ -898,11 +893,6 @@ void Logger::run()
 		}
 	}
 
-	int vehicle_status_sub = orb_subscribe(ORB_ID(vehicle_status));
-	uORB::SubscriptionData<parameter_update_s> parameter_update_sub(ORB_ID(parameter_update));
```

Comment:
Can you keep that here? It does not need to be a class attribute.

Reply:
Moved back.